src/app/organization/common/organization.service.ts

```diff
@@ -58,4 +58,9 @@ export class OrganizationService {
       )
   }
   
+  getUserOrganization(id: number): Observable<Response> {
+      return this.http.get(
+      `${organizationUrl}/user/${id}`
+      );
+  }
```

Comment:
for example, here we should use `rxjs` map operator:

```
return this.http.get(target_url).map(res => res.json());
```

and the return type can be `Observable<Organization>`